spaces/domains.json

```
@@ -23,6 +23,7 @@
   "gov.hyruleswap.com": "hyruleswap.eth",
   "complus.network": "complus.eth",
   "vote.axion.network": "axiondao.eth",
+  "vote.clucoin.com": "macbs.eth",
```

Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
  "vote.clucoin.com": "macsb.eth",
```

---

Somebody had purchased clu.eth and clucoin.eth before we decided to get an ENS address. Are you saying my ENS address must relate to `clucoin` as well? I own `getclucoin.eth` as well, would that ENS suffice?

---

Sorry I see what you are saying! I typoed

---

Should be fixed now